middlewares/addPrefix.go

```diff
@@ -11,7 +12,9 @@ type AddPrefix struct {
 }
 
 func (s *AddPrefix) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	r.URL.Path = s.Prefix + r.URL.Path
+	newPath := s.Prefix + r.URL.Path
+	log.Debugf("Adding Prefix %s to %s = %s", s.Prefix, r.URL.Path, newPath)
```

Comment:
Could you remove this Debug log? This is way too verbose to keep it in production code :)